docs/changes/coordinates/14763.feature.rst

```diff
@@ -0,0 +1,4 @@
+Support for ``_equatorial_radius`` and ``_flattening`` is added in
```

Comment:
I think we need a what's-new with an example of creating a geodetic subclass for some planet!